packages/logs/src/domain/logsCollection/networkError/networkErrorCollection.ts

```diff
@@ -27,12 +27,12 @@ export function startNetworkErrorCollection(configuration: LogsConfiguration, li
     }
   })
   const fetchSubscription = initFetchObservable().subscribe((context) => {
-    if (context.state === 'complete') {
+    if (context.state === 'resolve') {
       handleCompleteRequest(RequestType.FETCH, context)
     }
   })
 
-  function handleCompleteRequest(type: RequestType, request: XhrCompleteContext | FetchCompleteContext) {
+  function handleCompleteRequest(type: RequestType, request: XhrCompleteContext | FetchResolveContext) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`handleCompleteRequest` is not accurate anymore, maybe something like `handleResponse` would be enough